### PR TITLE
Add exec command to fda.

### PR DESCRIPTION
**Pull request by gz**
Execute a SQL query in adhoc mode and prints result on stdout.
Makes it easy to pipe it into a file and then diff two results.

### CODE REVIEW

File: crates/fda/src/cli.rs [hunk 1]
```diff
@@ -313,6 +313,35 @@ pub enum PipelineAction {
         #[arg(long, short = 's', default_value_t = false)]
         start: bool,
     },
+    /// Execute an ad-hoc query in a pipeline and return the result.
```
Comment (mihaibudiu) [inline crates/fda/src/cli.rs:316]: what do you mean "in a pipeline"?
What does this do?
Is this documented somewhere?

Comment (gz) [reply crates/fda/src/cli.rs]: updated comment, this is printed as part of --help

File: crates/fda/src/cli.rs [hunk 2]
```diff
@@ -313,6 +313,35 @@ pub enum PipelineAction {
         #[arg(long, short = 's', default_value_t = false)]
         start: bool,
     },
+    /// Execute an ad-hoc query against a pipeline and return the result.
+    Exec {
```
Comment (lalithsuresh) [inline crates/fda/src/cli.rs:317]: Better to call it "query"

Comment (gz) [reply crates/fda/src/cli.rs]: fixed

**Review by ryzhyk — APPROVED** [review]
Database CLIs usually allow executing a SQL script with many statements.
I guess this can be used to achieve the same effect by executing commands one by one. We may want to support that directly in the future.

**Pull request merged**